### PR TITLE
Improve UI consistency

### PR DESCRIPTION
**Pull request by jaucourt**
https://eaflood.atlassian.net/browse/FCEP-74

The pages in the app lack consistent HTML elements and application of
standards, such as presentation of selected options, using the same HTML
elements in the same way, etc.

### CODE REVIEW

**Comment by johnwatson484** [comment]
Not sure if the start page is in scope for these changes but here's a few observations.

image

I would remove all the "This is not a real service..." style warnings.  The existing demo service only mentions this as too many people thought we were prototyping a real mine support service.  Usually in a prototype you wouldn't have such warnings.  You design the prototype service as if it is 100% genuine as you want more accurate feedback on content and test assumptions with your design.  You don't want such warnings clouding that feedback.

The service name should be changed from "FFC Calculation Engine".  There is a lot of guidance around how to name a service which must be proven to be tied to user research when it comes to the GDS assessment, but the service should have a name that indicates what it's used for and what a user would be expecting to get out of it, rather than a brand name.  So you'd expect a service to be called something like "Apply for ELM", "Register to vote", "Renew your passport" etc.  A user isn't going to be searching online for "FFC Calculation Engine" as it means nothing to the public.

Here's a page that provides some guidance -> https://www.gov.uk/service-manual/design/naming-your-service

The start page should also inform the user what they should use this service for so it's clear what will happen if they click the start button and follow the journey.  Here is the associated guidance on start pages.  https://design-system.service.gov.uk/patterns/start-pages/

I'd also say the useful link isn't that useful.

**Comment by johnwatson484** [comment]
Again, not sure if this is in scope of this PR, but there is a bug in the parcel selection view as the error red line appears down the left side of the parcel list.

image

**Comment by johnwatson484** [comment]
Same bug on the fencing page with the red error line.

image

**Comment by johnwatson484** [comment]
image

This question is confusing as it doesn't mention which unit the user should enter.  Should it be metres, miles, inches etc.

I'd also maybe reword the question to be something like "how long is the fence you want to claim on in metres?"  Rather than a generic enter a fence length. You could also give an example of what you're expecting them to enter.  You can also make the field shorter.  This page has a good overview on text input fields -> https://design-system.service.gov.uk/components/text-input/

**Comment by johnwatson484** [comment]
image

I think this is good but I'd change the first title text to highlight to the user that they have finished the journey and there is nothing more they need to do.  So the example given in https://design-system.service.gov.uk/patterns/confirmation-pages/ is "Application Complete".  Then give the summary of what that means with regards to the payment.

As for the what happens next stage, if we are going to say that they need to make a formal application then we need to say how they do that.  But this makes me think that our journey in it's current state is a bit disjointed.  I think we need to return to the fundamental reason for this service's existence, what will our users use it for and this of course ties in with the name I mentioned in my previous comment.  My understanding (which may be wrong) is that users who want to apply for an FFC scheme want to know how much they are going to be paid if they submit their application.  So something like "Agricultural scheme calculator" (but probably not that) would be an appropriate service name similar to https://www.gov.uk/tax-credits-calculator.  We could also highlight in the before you start what information the user is likely to need before they click apply.

Incidentally here is how the Tax credits calculator ends it's journey.  
image  I'm not suggesting we copy this model as I think the confirmation page is clearer but we could take inspiration from it.  ie use the confirmation pattern to show the journey is ended and then highlight the monetary values underneath, then mention how they can claim based on this.  User research with actual users would usually help with the design but this is a bit of an unusual circumstance.

**Review by neilbmclaughlin — APPROVED** [review]
LGTM.

I've raised a separate PR for a validation issue I found (#12).
